test/rewardsTest.js

```diff
@@ -0,0 +1,147 @@
+const RewardsStub = artifacts.require("RewardsStub.sol")
+
+contract("Rewards", (accounts) => {
```

Comment:
In PR https://github.com/keep-network/sortition-pools/pull/127 we rewritten tests to `hardhat`.
So this test should be rewritten to hardhat as well (it should be easy and quick) after merging the `main` branch into this branch. Otherwise it won't work.

Reply:
Done in https://github.com/keep-network/sortition-pools/pull/130/commits/96aa93f01fba83cb7a2fafcb00b7ed5f221faddd. The conflict resolution was really painful.